cmake/onnxruntime_common.cmake

```diff
@@ -195,7 +195,7 @@ if (ARM64 OR ARM OR X86 OR X64 OR X86_64)
     # Its functionality in detecting x86 cpu features are lacking, so is support for Windows.
     if (CPUINFO_SUPPORTED)
       onnxruntime_add_include_to_target(onnxruntime_common cpuinfo::cpuinfo)
-      list(APPEND onnxruntime_EXTERNAL_LIBRARIES cpuinfo::cpuinfo)
+      list(APPEND onnxruntime_EXTERNAL_LIBRARIES cpuinfo::cpuinfo cpuinfo::clog)
```

Comment:
If pytorch_cpuinfo_SOURCE_DIR is not defined, I think cpuinfo::clog could be undefined too. If you build and install cpuinfo into your Linux system, will that installation also contain the clog lib? Will their cmake files export this cpuinfo::clog target? Unlikely they do. So your this change may break mayeut's .  I think it's ok to merge this PR first to get our pipelines back to normal. But we will still need to find a better solution.

Reply:
> Will their cmake files export this cpuinfo::clog target?

is that handled here?
https://github.com/pytorch/cpuinfo/blob/eb4a6674bfe9cf91b63b9817412ae5f6862c8432/deps/clog/CMakeLists.txt#L72-L76

Reply:
Yes. Thanks!